core/lib/vm/src/tracers/traits.rs

```diff
@@ -29,6 +27,8 @@ pub trait ExecutionProcessing<S: WriteStorage, H: HistoryMode>:
         _stop_reason: VmExecutionStopReason,
     ) {
     }
+
+    fn save_results(&mut self, _result: &VmExecutionResultAndLogs) {}
```

Comment:
Same here - AFAIK you mentioned that you are planning to remove this one very soon, right? or does it stay?

Reply:
Yeah, and i've removed it completely..